pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -28,59 +24,17 @@ const (
 	ProjectTagCount             int = 1
 )
 
-type SecurityPolicyService struct {
+type Service struct {
+	common.Service
 	Client              client.Client
-	NSXClient           *nsx.Client
-	NSXConfig           *config.NSXOperatorConfig
 	GroupStore          cache.Indexer
 	SecurityPolicyStore cache.Indexer
 	RuleStore           cache.Indexer
 }
 
-var log = logf.Log.WithName("service").WithName("firewall")
-
-// InitializeSecurityPolicy sync NSX resources
-func InitializeSecurityPolicy(NSXClient *nsx.Client, cf *config.NSXOperatorConfig) (*SecurityPolicyService, error) {
```

Comment:
As we discussed in the patch: 
https://github.com/vmware-tanzu/nsx-operator/pull/119/files#diff-00dd8c9577ada3e37b0c0c62c064c77de265017aa22a33e2b506ef6ab2ea8e81

please keep this func in firewall.go.

Reply:
ok